nilearn/connectome/group_sparse_cov.py

```diff
@@ -973,29 +1095,46 @@ def fit(self, subjects, y=None):
             # Compute the cross-validated loss on the current grid
             train_test_subjs = []
             for train_test in this_cv:
-                assert(len(train_test) == n_subjects)
-                train_test_subjs.append(list(zip(*[(subject[train, :],
-                                                    subject[test, :])
-                                             for subject, (train, test)
-                                             in zip(subjects, train_test)])))
+                assert len(train_test) == n_subjects
+                train_test_subjs.append(
+                    list(
+                        zip(
+                            *[
+                                (subject[train, :], subject[test, :])
+                                for subject, (train, test) in zip(
+                                    subjects, train_test
+                                )
+                            ]
+                        )
+                    )
+                )
```

Comment:
Do we want to make this a bit easier to parse for humans?

Reply:
I agree it's not pretty at all.... I would have extracted this in a separate function but that is beyond the formatting scope of this PR.
If you have an idea of a more readable formatting then please go for it, otherwise I think we can leave it like this and refactor later.

Reply:
Good point.
I think I have actually flagged this function on my repo as a good candidate for refactoring, so let's cross that bridge when we get to it.
https://github.com/Remi-Gau/nilearn/issues/19